src/components/Logo/AssetLogo.tsx

```diff
@@ -0,0 +1,60 @@
+import { SupportedChainId } from 'constants/chains'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { MissingImageLogo } from './BaseLogo'
+
+const LogoImage = styled.img<{ size: string }>`
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  background: radial-gradient(white 60%, #ffffff00 calc(70% + 1px));
+  border-radius: 50%;
+  -mox-box-shadow: 0 0 1px white;
+  -webkit-box-shadow: 0 0 1px white;
+  box-shadow: 0 0 1px white;
+`
+
+export type AssetLogoBaseProps = {
+  symbol?: string | null
+  backupImg?: string | null
+  print?: boolean
```

Comment:
What is `print` for? This should be commented.

Reply:
meant to take that out, thx for catching